### PR TITLE
Add guidelines for workshops

### PR DESCRIPTION
**Pull request by Alfriadox**

### CODE REVIEW

**Comment by Apexal** [comment]
This is awesome and much needed.

**Review by Apexal — APPROVED** [review]

File: docs/events/workshops.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Workshops
+
+Workshops are an important part of the RCOS experience. They strengthen the presenters communication skills and inform the audience about a skill or technology 
+pertinent to their open source work. 
+
+## Hosting a workshop
+Anyone in RCOS is allowed to host a workshop. Mentors are required to host at least one. 
+
+Workshops have historically be hosted mostly on Wednesdays, since most Computer Science majors at RPI are free on Wednesdays afternoon. Despite this,
```
Comment (hbh7) [inline docs/events/workshops.md:9]: "Wednesdays afternoon" should probably be "Wednesday afternoons"

File: docs/events/workshops.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+## Topic Requirements
+It is required that workshops are pertinent to open source software or the open source community. This is more flexible than many people might expect. 
+
+In general, all workshops that teach a skill are acceptable. This can be anything from building a PC to updating your resume to creating an effective presentation. 
+The reason for this is that these are all skills that will help people in the open source community (and by extension in RCOS) do their work or get a job. It's 
+mostly difficult to find a skill that would not be useful to open source developers. If you are in doubt or have questions though, feel free to reach out to a 
+coordinator or faculty advisor. 
+
+Workshops that focus on a specific technology or protocol should be sure that the technology is open source. There are some exceptions, 
+for example git is open source but GitHub is not. Despite being closed source, GitHub (or parts of it like GitHub Actions) would obviously be an acceptable
```
Comment (hbh7) [inline docs/events/workshops.md:22]: Git should probably be capitalized

**Review by hbh7 — CHANGES_REQUESTED** [review]
Looks good to me! I just found some minor grammatical things that we might want to change, that's about it.

File: docs/events/workshops.md (hunk 1 above)
Comment (Alfriadox) [reply docs/events/workshops.md]: Fixed!

File: docs/events/workshops.md (hunk 2 above)
Comment (Alfriadox) [reply docs/events/workshops.md]: Fixed!

**Review by hbh7 — APPROVED** [review]
:D

**Pull request merged**